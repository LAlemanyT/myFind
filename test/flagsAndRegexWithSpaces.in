./myfind.py ~ --regex=" " "cat {} -n"  
