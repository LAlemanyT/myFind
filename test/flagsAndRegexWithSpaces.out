cat: '/home/testy test': Is a directory
1  #!/bin/bash
2
3  echo hi
cat: '/home/testy test/testy test': Is a directory
